composer.json

```diff
@@ -76,7 +76,7 @@
         "iron-io/iron_mq": "~2.0",
         "mockery/mockery": "~0.9.2",
         "pda/pheanstalk": "~3.0",
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "~4.1",
```

Comment:
Sure, I'll submit one later. The conversation on that got collapsed from the rebase. There are method missing in 4.0 that a test needs.